### ISSUE TITLE
Crash on left mouse button hold and window exit

### ISSUE THREAD

**Issue by wilsonk** [JonSnowbd/ZT#5]
Hello,

I was playing around with your great library, just drawing some lines/polyline figures, etc. and I found that if I held the left mouse button down and left the window surface, out the left side of the window, then the example programs crash. This happens even if the example doesn't necessarily 'draw' during the left mouse button hold (so the imgui example, for instance).

Seems like it must be an OpenGL error as zig doesn't capture any errors and display a stack trace. The program just hard exits. Only tested on Windows. Sometimes you have to move the mouse pretty aggressively to get  the crash. It doesn't happen in the other three directions, even though I have a triple monitor setup and drag all the way onto the other monitors.

Just thought I would point it out. Not a show stopper at all, just a little annoying when it happens :)

**Comment by wilsonk** [comment]
I built the examples with a forced debugging option and attached VS2019 to it (gdb seems to be broken on my MSYS2 setup?)...anyways, the program still fails the same way with no stack trace and VS just says it exited normally. Strange. Just thought I would let you know.

I haven't tried this on my Linux machines yet.

**Comment by JonSnowbd** [comment]
I cant replicate this on my own desktop. You mention vs2019, are you building specifically for MSVC in zig with say `-Dtarget=native-windows-msvc`? Does this happen on release builds?

**Comment by wilsonk** [comment]
Woops, sorry to bother JonSnowbd, I just figured this out. I have a program called barrier running to share my mouse with the third screen on the left and that is what is causing the crash. Interesting, but totally my fault, and I apologize for taking any of your time.

Just so you know it happens in all modes, and I wasn't building with any changes to the build file (except if I forced debug, just to be sure the symbols were there)...but really this is irrelevant, I suppose :)
